bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -414,9 +418,19 @@ spec:
   - Kuadrant
   links:
   - name: Kuadrant Operator
-    url: https://kuadrant-operator.domain
+    url: https://github.com/Kuadrant/kuadrant-operator
+  - name: Kuadrant Docs
+    url: https://kuadrant.io
+  maintainers:
+  - email: eastizle@redhat.com
+    name: Eguzki Astiz Lezaun
+  - email: mcassola@redhat.com
+    name: Guilherme Cassolato
+  - email: didier@redhat.com
+    name: Didier Di Cesare
   maturity: alpha
+  minKubeVersion: 1.8.0
   provider:
     name: Red Hat
     url: https://github.com/Kuadrant/kuadrant-operator
-  version: 0.0.0
+  version: 0.1.0
```

Comment:
I think that in `main` branch, we should be using `0.0.0` for the version and `latest` for the image tags

Reply:
> I think that in `main` branch, we should be using `0.0.0` for the version and `latest` for the image tags

@eguzki's suggestion matches what we do in https://github.com/Kuadrant/authorino-operator/blob/7e4022a92e6859a18765377d26fd4d0d88bd37b5/Makefile#L24-L25

Reply:
Makes sense